hiera-eyaml-gpg.gemspec

```diff
@@ -17,5 +17,4 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
 
   gem.add_dependency('hiera-eyaml', '>=1.3.8')
-  gem.add_dependency('puppet', '>=5.5.8')
```

Comment:
Should this have been a development dependency?

Reply:
Maybe.  For now I just want to get a non-borked release out that people can install.

Reply:
I also don't know about `gpgme` and `ruby_gpg`.  When testing, you need one or the other (but not both) in your Gemfile.  I guess some default with an environment variable to switch between the 2??  It's not super important until someone finds the time, inclination and skill to write some sort of test suite.